guides/common/modules/proc_adding-an-http-proxy.adoc

```diff
@@ -14,17 +14,7 @@ To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-an-http-pr
 . In the *URL* field, enter the URL for the HTTP proxy, including the port number.
 The following host names are available:
 +
-[cols="2,1,1",options="header"]
-|====
-| Host name | Port | Protocol
-| subscription.rhsm.redhat.com | 443 | HTTPS
-| subscription.rhn.redhat.com | 443 | HTTPS
-| cdn.redhat.com | 443 | HTTPS
-ifdef::satellite[]
-| api.access.redhat.com (if using Red{nbsp}Hat Insights) | 443 | HTTPS
-| cert-api.access.redhat.com (if using Red{nbsp}Hat Insights) | 443 | HTTPS
-endif::[]
-|====
+include::snip_host-names-for-http-proxy.adoc[]
```

Comment:
The table placement is still wrong in this file.

Reply:
From the procedure, it still looks like the table is placed properly.
Also, I speculated there is no change in placement from your last comment. :sweat_smile: 

Can you guide me to the desired placement?

Reply:
Hah. I think I managed to confuse myself and then you. Sorry! Let me see if I can get the things straight.

The table of "hosts the proxy needs to allow" currently exists in two files:
* `guides/common/modules/proc_adding-an-http-proxy.adoc`
* `guides/common/modules/proc_configuring-http-proxy-to-connect-to-cdn.adoc`

The original issue was: "`proc_adding-an-http-proxy.adoc` is missing the `akamaiedge.net` entry" and while reviewing your change to that, I noticed the wrong placement and asked you to move the table to a snippet and include that snippet in both files, fixing the placement.

And then, you asked whether you should add it to both files, I accidentally said "I'd replace the table in `proc_adding-an-http-proxy.adoc` with an include, but that's it." which was wrong, I meant the "that's it part" to `proc_configuring-http-proxy-to-connect-to-cdn.adoc` as that's the file that we were originally not touching in this PR.

So in regard with `proc_adding-an-http-proxy.adoc` I'd add a "Prerequisites" section, reading something like this:
```
Your HTTP proxy must allow access to the following hosts:
include::snip_host-names-for-http-proxy.adoc[]
```

Reply:
Thank you for the detailed clarification @evgeni 
It's perfectly okay for the confusion (usually you are always clear). 😀

I hope, we inched closer now.